__tests__/app.test.js

```diff
@@ -122,3 +122,23 @@ describe("patch api articles", () => {
       });
   });
 });
+
+describe("get api users", () => {
+  test("200: should return an array of users with their usernames", () => {
```

Comment:
I'd recommend testing a bit more throughly here. Maybe for a loop and checking that all the expected keys are on each user :)